test/parallel_testsuite.py

```diff
@@ -130,17 +132,17 @@ def stopTest(self, test):
 
   def addSuccess(self, test):
     if hasattr(time, 'perf_counter'):
-      print(test, '... ok (%.2fs)' % (self.calculateDuration()), file=sys.stderr)
+      print(test, '... ok (%.2fs)' % (self.calculateElapsed()), file=sys.stderr)
```

Comment:
What is this different to reporting `self.test_duration` ?

Reply:
It is an processing order thing with Python's test harness. `self.test_duration` has not been calculated yet at this point, i.e. `BufferedParallelTestResult.addDuration()` has not yet been called by the time this function is called, so `self.test_duration` would still report zero.